collector/systemd_linux.go

```diff
@@ -27,7 +27,7 @@ import (
 
 var (
 	unitWhitelist = flag.String("collector.systemd.unit-whitelist", ".+", "Regexp of systemd units to whitelist. Units must both match whitelist and not match blacklist to be included.")
-	unitBlacklist = flag.String("collector.systemd.unit-blacklist", "", "Regexp of systemd units to blacklist. Units must both match whitelist and not match blacklist to be included.")
+	unitBlacklist = flag.String("collector.systemd.unit-blacklist", ".+\\.scope", "Regexp of systemd units to blacklist. Units must both match whitelist and not match blacklist to be included.")
```

Comment:
We generally have anchored regexes everywhere in Prometheus, and the existing one above also assumes that. I'd be for leaving it as is...